packs/nest/actions/humidity.py

```diff
@@ -0,0 +1,14 @@
+from lib import actions
+
+class HumidityAction(actions.BaseAction):
+    def run(self, structure, device, target=None):
+        if target:
+            target_humidity = target
+        else:
+            target_humidity = self._get_device(structure, device).humidity
```

Comment:
Hm, does this case (not providing a target) even makes sense?

It looks like it just synchronizes all the devices to same target of the selected device?

If my observation is correct and this case still makes sense, we should document this in the parameter description.